kytos/core/auth.py

```diff
@@ -44,6 +50,83 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class UserController:
+    """UserController"""
+
+    # pylint: disable=unnecessary-lambda
+    def __init__(self, get_mongo=lambda: Mongo()):
+        self.mongo = get_mongo()
+        self.db_client = self.mongo.client
+        self.db = self.db_client[self.mongo.db_name]
+
+    def bootstrap_indexes(self):
+        """Bootstrap all topology related indexes."""
+        index_tuples = [
+            ("users", [("username", pymongo.ASCENDING)]),
```

Comment:
We need `users.username` to also be an unique index, let's set it to be unique.

Reply:
Done. I realized that I had to drop the collection first for it to work.